Readme.md

```diff
@@ -1,89 +1,95 @@
 # ezl: easyLambda
-> Parallel dataflows for easy data processing with functional, C++ and MPI
+> Parallel data processing made easy using dataflows in modern C++
```

Comment:
How about "Parallel programming with dataflow and functional programming in modern C++" or "Parallel data processing with dataflows and functional in modern C++"